darwin/future/core/items/move_items.py

```diff
@@ -31,7 +31,7 @@ def move_items_to_stage(
 
     Returns
     -------
-    None
+    JSONType
```

Comment:
This and the other file change are just two minor changes to test that it was grabbing the right files